src/mixins/nft.js

```diff
@@ -352,6 +355,16 @@ export default {
           })
         : [defaultEdition];
     },
+    nftBookFeaturedPrice() {
```

Comment:
I don't think this is a good name
A nft book can have fiat price without being featured
A featured book can be sold out in fiat but listed on market with LIKE